airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DbtTransformationRunner.java

```diff
@@ -99,6 +99,7 @@ public boolean transform(final String jobId,
               jobRoot,
               dbtConfig.getDockerImage(),
               false,
```

Comment:
@davinchia or @mfsiega-airbyte I don't have much context on this one - from the comment it seems transformation will only be provided by user, but for custom connector are we allowing them to use Dbt related features?

Reply:
We don't currently support custom DBT transformations in Cloud (and we deprioritized the work to enable custom transformations on kube this quarter) but I think DBT transformations are only supported in a specific list of destinations, just like with normalization. So for custom destination connectors, I don't think we need to worry about DBT transformations or Normalization (@davinchia should confirm though to be sure)

Reply:
yeap, Parker is correct here too!